frontend/src/APIClients/CommonAPIClient.ts

```diff
@@ -93,8 +94,31 @@ const getUserStatus = async (email: string): Promise<string> => {
   }
 };
 
+const createResident = async ({
```

Comment:
it's fine to keep this here for now, but it will need to be moved into a `ResidentsAPIClient` file once #129 is merged